docs/products/kafka/howto/public-access-in-vpc.rst

```diff
@@ -0,0 +1,14 @@
+Enable public access in a VPC
+==============================
+
+To enable public access for **Aiven for Apache Kafka** service which is running within a virtual private cloud (VPC), follow these steps:
+
+#. Log in to the Aiven web console and select your **Aiven for Apache Kafka** service.
+#. On the *Overview* page, scroll down to the *Advanced configuration* section and click **Add configuration option** .
+#. Select ``public_access.kafka`` and switch it on. For **Aiven for Apache Kafka Connect** or **Aiven for Apache Kafka Rest**, select ``public_access.kafka_connect`` or ``public_access.kafka_rest`` respectively.
```

Comment:
"Aiven for Apache Kafka Rest" is not a product name I recognise. Maybe replace with "Karapace REST API" or "Karapace REST Proxy" ?